web/src/features/fireWeather/components/graphs/TempRHGraph.tsx

```diff
@@ -15,25 +15,25 @@ import {
   rangeSliderConfig
 } from 'features/fireWeather/components/graphs/plotlyHelper'
 
-const observedTempColor = '#fb0058'
-const observedRHColor = '#005f87'
+const observedTempColor = '#4291FF'
+const observedRHColor = '#EB5757'
 const observedDewpointColor = '#02fa44'
-const forecastTempColor = '#a50b41'
-const forecastRHColor = '#17c4c4'
-const hrdpsTempColor = '#a017c2'
-const hrdpsRHColor = '#3ac417'
+const forecastTempColor = '#1200DE'
+const forecastRHColor = '#910000'
+const hrdpsTempColor = '#B86BFF'
+const hrdpsRHColor = '#229B56'
 const hrdpsTempPlumeColor = 'rgba(203, 169, 214, 0.4)'
 const hrdpsRHPlumeColor = 'rgba(181, 240, 165, 0.4)'
-const rdpsTempColor = '#ea6d0e'
-const rdpsRHColor = '#026200'
+const rdpsTempColor = '#FF64DD'
+const rdpsRHColor = '#285777'
 const rdpsTempPlumeColor = 'rgba(244, 143, 65, 0.4)'
 const rdpsRHPlumeColor = 'rgba(42, 137, 137, 0.4)'
-const gdpsTempColor = '#f56c9c'
-const gdpsRHColor = '#32e7e7'
+const gdpsTempColor = '#7556CA'
+const gdpsRHColor = '#F2994A'
 const gdpsTempPlumeColor = 'rgba(255, 150, 169, 0.4)'
 const gdpsRHPlumeColor = 'rgba(148, 255, 235, 0.4)'
```

Comment:
with Dev down in OCP4 it's kinda hard to tell what's going on, but I wanted to flag this just in case - I would have expected that the plume colours would also be updated? Is that what the squares labelled "20%" are for @ambroseyau ? Also, was there a new colour for observed Dewpoint? I'm just seeing a blank square for that in Figma

Reply:
@andrea-williams dewpoint color change incoming. Not sure about the plume colors...

Reply:
The 20% is the color for the area color underneath the line graphs. The reason why we had to use the plume color is because users would be able to tell these are the model colours. Also, In most cases, users will only open all models at once to see if they're aligned. Chances of users using all models at once is quite infrequent.